commons/protocol/src/test/resource/saved-protocol/0.2/complex/moduleid.json

```diff
@@ -0,0 +1 @@
+{"$type":"sbt.protocol.ModuleId","attributes":{"$type":"scala.collection.immutable.Map[java.lang.String,java.lang.String]","a":"b"},"name":"bar","organization":"com.foo"}
```

Comment:
I need it for the "T".  If you want this to be an "obj" you have to deal with not being able to use "$type" as a key....